Lib/unittest/test/test_assertions.py

```diff
@@ -106,7 +106,6 @@ class A:
         wr = None
 
         class Foo(unittest.TestCase):
-
```

Comment:
Core team probably would reject this because formatting changes aren't accepted: https://devguide.python.org/pullrequest/?highlight=formatting#making-good-prs

Reply:
That's correct.